### ISSUE TITLE
Error: 'persian' cannot be parsed to a Locale

### ISSUE THREAD

**Issue by hamid914** [caskroom/homebrew-cask#34722]
#### General troubleshooting steps

- [x] I have checked the instructions for [reporting bugs](https://github.com/caskroom/homebrew-cask#reporting-bugs) (or [making requests](https://github.com/caskroom/homebrew-cask#requests)) before opening the issue.
  - [x] None of the templates was appropriate for my issue, or I’m not sure.
- [x] I ran `brew update-reset && brew update` and retried my command.
- [x] I ran `brew doctor`, fixed as many issues as possible and retried my command.
- [x] I understand that [if I ignore these instructions, my issue may be closed without review](https://github.com/caskroom/homebrew-cask/blob/master/doc/faq/closing_issues_without_review.md).

#### Description of issue

I am using brew cask in Zsh-5.3.1 on tmux-2.4 running in iTerm2-3.0.15. It was running fine until yesterday. I run these commands every day:
```
brew upgrade
brew doctor
brew cask outdated | xargs brew cask reinstall
```
Yesterday, the last line produced these errors:
```
Error: 'persian' cannot be parsed to a Locale
Follow the instructions here:
  https://github.com/caskroom/homebrew-cask#reporting-bugs
/usr/local/Homebrew/Library/Homebrew/locale.rb:15:in `parse'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/dsl.rb:126:in `map'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/dsl.rb:126:in `language_eval'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/dsl.rb:115:in `language'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cask_loader.rb:107:in `block in load'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cask.rb:17:in `instance_eval'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cask.rb:17:in `initialize'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cask_loader.rb:51:in `new'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cask_loader.rb:51:in `cask'
/usr/local/Homebrew/Library/Homebrew/compat/hbc/cask_loader.rb:10:in `cask'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cask_loader.rb:9:in `load'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cask_loader.rb:9:in `instance_eval'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cask_loader.rb:9:in `load'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cask_loader.rb:41:in `load'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cask_loader.rb:128:in `load'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/scopes.rb:38:in `block in installed'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/scopes.rb:30:in `map'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/scopes.rb:30:in `installed'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli/list.rb:63:in `list_installed'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli/list.rb:14:in `run'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli.rb:115:in `run_command'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli.rb:158:in `process'
/usr/local/Homebrew/Library/Homebrew/cmd/cask.rb:8:in `cask'
/usr/local/Homebrew/Library/Homebrew/brew.rb:93:in `<main>'
```
I ran `brew update-reset && brew update` but errors exist yet. There are locale settings in my `.zshrc`:
```
export LANG=en_US.UTF-8
export LC_ALL=en_US.UTF-8
```
My macOS main system language is Persian. I changed that default to English but the problem persists.

#### Output of your command with `--verbose --debug`

```
$ brew cask ls --verbose --debug
brew cask ls --verbose --debug
Error: 'persian' cannot be parsed to a Locale
Follow the instructions here:
  https://github.com/caskroom/homebrew-cask#reporting-bugs
/usr/local/Homebrew/Library/Homebrew/locale.rb:15:in `parse'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/dsl.rb:126:in `map'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/dsl.rb:126:in `language_eval'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/dsl.rb:115:in `language'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cask_loader.rb:107:in `block in load'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cask.rb:17:in `instance_eval'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cask.rb:17:in `initialize'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cask_loader.rb:51:in `new'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cask_loader.rb:51:in `cask'
/usr/local/Homebrew/Library/Homebrew/compat/hbc/cask_loader.rb:10:in `cask'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cask_loader.rb:9:in `load'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cask_loader.rb:9:in `instance_eval'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cask_loader.rb:9:in `load'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cask_loader.rb:41:in `load'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cask_loader.rb:128:in `load'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/scopes.rb:38:in `block in installed'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/scopes.rb:30:in `map'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/scopes.rb:30:in `installed'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli/list.rb:63:in `list_installed'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli/list.rb:14:in `run'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli.rb:115:in `run_command'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli.rb:158:in `process'
/usr/local/Homebrew/Library/Homebrew/cmd/cask.rb:8:in `cask'
/usr/local/Homebrew/Library/Homebrew/brew.rb:93:in `<main>'
Error: Kernel.exit
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli.rb:169:in `exit'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli.rb:169:in `rescue in process'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli.rb:149:in `process'
/usr/local/Homebrew/Library/Homebrew/cmd/cask.rb:8:in `cask'
/usr/local/Homebrew/Library/Homebrew/brew.rb:93:in `<main>'
```

#### Output of `brew cask doctor`

```
$ brew cask doctor
brew cask doctor
==> Homebrew-Cask Version
Homebrew-Cask 1.2.1-40-gdd5d4880
caskroom/homebrew-cask (git revision e35348; last commit 2017-05-22)
==> Homebrew-Cask Install Location
<NONE>
==> Homebrew-Cask Staging Location
/usr/local/Caskroom
==> Homebrew-Cask Cached Downloads
~/Library/Caches/Homebrew/Cask (47 files, 1.4GB)
==> Homebrew-Cask Taps:
/usr/local/Homebrew/Library/Taps/caskroom/homebrew-cask (3615 casks)
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-apache (0 casks)
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core (0 casks)
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-dupes (0 casks)
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-games (0 casks)
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-php (0 casks)
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-python (0 casks)
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-science (0 casks)
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-services (0 casks)
/usr/local/Homebrew/Library/Taps/neovim/homebrew-neovim (0 casks)
/usr/local/Homebrew/Library/Taps/universal-ctags/homebrew-universal-ctags (0 casks)
==> Contents of $LOAD_PATH
/usr/local/Homebrew/Library/Homebrew/cask/lib
/usr/local/Homebrew/Library/Homebrew
/Library/Ruby/Site/2.0.0
/Library/Ruby/Site/2.0.0/x86_64-darwin16
/Library/Ruby/Site/2.0.0/universal-darwin16
/Library/Ruby/Site
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/vendor_ruby/2.0.0
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/vendor_ruby/2.0.0/x86_64-darwin16
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/vendor_ruby/2.0.0/universal-darwin16
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/vendor_ruby
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/x86_64-darwin16
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/universal-darwin16
==> Environment Variables
LANG="en_US.UTF-8"
LC_ALL="en_US.UTF-8"
LC_CTYPE="”en_US.UTF-8″"
PATH="/usr/local/sbin:~/.composer/vendor/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/opt/X11/bin:/usr/local/MacGPG2/bin:/Library/TeX/texbin:~/.fzf/bin:/usr/local/Homebrew/Library/Taps/homebrew/homebrew-services/cmd:/usr/local/Homebrew/Library/Homebrew/shims/scm"
SHELL="/usr/local/bin/zsh"
```

**Comment by reitermarkus** [comment]
What does `defaults read .GlobalPreferences AppleLanguages` return?

**Comment by hamid914** [comment]
It returns:
```
(
    "fa-IR",
    "persian [common]-IR",
    "en-IR"
)
```
As the same as settings, I set in "Language and Region" of System Preferences.

**Comment by hamid914** [comment]
I had installed `cocoaspell` from homebrew-cask with this command:
```
brew cask install cocoaspell
```
Then I had imported [Farsi](ftp://ftp.gnu.org/gnu/aspell/dict/fa/aspell6-fa-0.11-0.tar.bz2) language in `/Library/Application\ Support/cocoAspell` and then compiled language in System Preferences > Spelling. After that Cocoaspell added "persian [common]-IR" language to my system languages (3rd line in the previous comment).
I removed that language from "Language and Region" and then brew cask worked fine!
Thanks for your help.
